### PR TITLE
Fix error in jaeger test

### PR DESCRIPTION
**Pull request by josunect**
Fix error https://github.com/kiali/kiali/actions/runs/6316367574/job/17151097502
```
TASK [Assert that there is an error related to Jaeger] *************************
fatal: [localhost]: FAILED! => {
    "assertion": "jaeger_error is defined",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}
```

Ref. https://github.com/kiali/kiali/issues/6537

### CODE REVIEW

File: molecule/jaeger-test/converge.yml [hunk 1]
```diff
@@ -25,14 +25,14 @@
   # If Jaeger is in the list, it means that there is an error (Unreacheable)
   - name: Check if Jaeger status is present
     set_fact:
-      jaeger_error: "{{ item.status }}"     
+      tracing_error: "{{ item.status }}"
     loop: "{{ status_response.json }}"
     when: "item.name == 'jaeger'"
```
Comment (jmazzitelli) [inline molecule/jaeger-test/converge.yml:30]:
```suggestion
    when: "item.name == 'tracing'"
```

File: molecule/jaeger-test/converge.yml [hunk 2]
```diff
@@ -57,11 +57,11 @@
 
   - name: Check if Jaeger status is present
     set_fact:
-      jaeger_error: "{{ item.status }}"     
+      tracing_error: "{{ item.status }}"
     loop: "{{ status_response.json }}"
     when: "item.name == 'jaeger'"
```
Comment (jmazzitelli) [inline molecule/jaeger-test/converge.yml:62]:
```suggestion
    when: "item.name == 'tracing'"
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
Changing the ansible variable names doesn't actually fix anything :)

See my suggested changes that should fix the tests.

**Comment by josunect** [comment]
> Changing the ansible variable names doesn't actually fix anything :)
> 
> See my suggested changes that should fix the tests.

Oh, I should have read a bit more about ansible first ... I thought it was related to a type that is named also jaeger_error and was renamed...

**Review by jmazzitelli — APPROVED** [review]
I haven't had a change to run this, but this looks correct.

We'll know it works when the  CI "Test Molecule" action runs ;)

**Comment by josunect** [comment]
> I haven't had a chance to run this, but this looks correct.
> 
> We'll know it works when the CI "Test Molecule" action runs ;)

So, I'll merge then :)

**Pull request merged**